README.rst

```diff
@@ -202,11 +202,12 @@ with easily recognisable names. For example:
     >>> Lambda_b_0.J
     0.5
 
-You can quickly search for particles from the command line with:
+You can quickly search for particles from the command line with
+(note: quotes may be used/needed but only double quotes work as expected on Windows):
 
 .. code-block:: bash
 
-    $ python -m particle search 'K*0'
+    $ python -m particle search K*0
```

Comment:
Let's use double quotes here. The only general danger with double quotes is that shell variables are expanded, but no particles have `$` in their name so this should be fine.